packages/js/src/components/WincherRankingHistoryChart.js

```diff
@@ -0,0 +1,109 @@
+/* External dependencies */
+import { Line } from "react-chartjs-2";
+import { CategoryScale, Chart, LineController,	LineElement, LinearScale, PointElement, TimeScale, Colors, Legend, Tooltip } from "chart.js";
+import "chartjs-adapter-moment";
+import PropTypes from "prop-types";
+import { noop } from "lodash";
+import moment from "moment";
+
+Chart.register( CategoryScale, LineController,	LineElement, PointElement, LinearScale, TimeScale, Colors, Legend, Tooltip );
+
+/**
+ * Renders the Wincher ranking history chart.
+ *
+ * @param {Object} props The ranking history props.
+ *
+ * @returns {null|wp.Element} The Wincher ranking history chart.
+ */
+export default function WincherRankingHistoryChart( { datasets, isChartShown } ) {
+	if ( ! isChartShown ) {
+		return null;
+	}
+
+	const data = datasets.map( dataset => ( {
+		...dataset,
+		data: dataset.data.map( ( { datetime, value } ) => ( {
+			x: datetime,
+			y: value,
+		} ) ),
+		lineTension: 0,
+		pointRadius: 0,
+		pointHoverRadius: 2,
+		borderWidth: 2,
+	} ) );
+
+	return (
+		<Line
+			height={ 100 }
+			data={ {
+				datasets: data,
+			} }
+			options={ {
+				plugins: {
+					legend: {
+						display: true,
+						position: "bottom",
+						labels: {
+							color: "black",
+							usePointStyle: true,
+							boxHeight: 7,
+							boxWidth: 7,
+						},
+						onClick: noop,
+					},
+					tooltip: {
+						enabled: true,
+						callbacks: {
+							title: ( x ) => moment( x[ 0 ].label ).format( "YYYY-MM-DD" ),
```

Comment:
I we need to ensure this prints the date in GMT as otherwise it'll be different from the Wincher UI

Reply:
Can you confirm how it works now? Does it print based on local tz?

Reply:
Yes it displays date + 2AM

Reply:
I think best would be to make it GMT for now..?

Reply:
Should be fixed